source/numerics/kdtree.cc

```diff
@@ -67,7 +67,7 @@ KDTree<dim>::get_closest_points(const Point<dim> & target,
   std::vector<unsigned int> indices(n_points);
   std::vector<double>       distances(n_points);
 
-  kdtree->knnSearch(&target[0], n_points, &indices[0], &distances[0]);
+  kdtree->knnSearch(&target[0], n_points, indices.data(), distances.data());
```

Comment:
You could probably use
```suggestion
  kdtree->knnSearch(target.begin_raw(), n_points, indices.data(), distances.data());
```
or did you avoid accessing raw memory here purposefully?

Reply:
In line 51 as well